src/extensions/default/DebugCommands/main.js

```diff
@@ -257,6 +272,461 @@ define(function (require, exports, module) {
         brackets.app.showOSFolder(dir);
     }
 
+    function _getDefaultPrefsFilePath() {
+        // Default preferences
+        return defaultPreferencesFullPath;
+    }
+
+    function _openPrefFilesInSplitView(prefsPath, defaultPrefsPath) {
+
+        var currScheme       = MainViewManager.getLayoutScheme(),
+            file             = FileSystem.getFileForPath(prefsPath),
+            defaultPrefsFile = FileSystem.getFileForPath(defaultPrefsPath);
+
+        var resultObj = MainViewManager.findInAllWorkingSets(defaultPrefsPath);
+        if (resultObj && resultObj.length > 0) {
+            CommandManager.execute(Commands.FILE_CLOSE, {file: defaultPrefsFile, paneId: resultObj[0].paneId});
+        }
+
+        // Open the default preferences in the left pane in the read only mode.
+        CommandManager.execute(Commands.FILE_OPEN, { fullPath: defaultPrefsPath, paneId: "first-pane", options: { isReadOnly: true } });
+
+        if (currScheme.rows === 1 && currScheme.columns === 1) {
+            // Split layout is not active yet. Inititate the
+            // split view.
+            MainViewManager.setLayoutScheme(1, 2);
+        }
+
+
+        // Make sure the preference file is going to be opened in the second
+        // pane
+        if (MainViewManager.findInWorkingSet("first-pane", prefsPath) >= 0) {
+
+            MainViewManager._moveView("first-pane", "second-pane", file, 0, true);
+
+            // Now refresh the project tree by asking
+            // it to rebuild the UI.
+            WorkingSetView.refresh(true);
+        }
+
+        CommandManager.execute(Commands.FILE_OPEN, { fullPath: prefsPath, paneId: "second-pane"});
+
+    }
+
+    function _isSupportedPrefType(prefType) {
+        if (prefType === "number" ||
+                prefType === "boolean" ||
+                prefType === "string"  ||
+                prefType === "array"   ||
+                prefType === "object") {
```

Comment:
I think we should use an array of supported preference types here and just check whether or not `prefType` is in it.